gateway/src/main/java/org/georchestra/gateway/filter/headers/RemoveHeadersGatewayFilterFactory.java

```diff
@@ -61,10 +71,20 @@
  * 
  */
 @Slf4j(topic = "org.georchestra.gateway.filter.headers")
+@EnableConfigurationProperties(LdapConfigProperties.class)
 public class RemoveHeadersGatewayFilterFactory extends AbstractGatewayFilterFactory<RegExConfig> {
 
-    public RemoveHeadersGatewayFilterFactory() {
+    @Autowired
+    private Environment environment;
+
+    @Value("${georchestra.gateway.security.createNonExistingUsersInLDAP:}")
+    private String trusted;
```

Comment:
the variable name does not seem to match the one coming from the configuration file

Reply:
resolving as outated